### DISCUSSION TITLE
Pandas to Vaex cheatsheet

### DISCUSSION THREAD

**Discussion by MichalChromcak** [vaexio/vaex#2056] [Ideas]
Hi team!

I really love what you are up to with vaex and want to express my gratitude for having it. 🚀  

Implementing it on the codebase I need, I though feel that I spent a considerable amount of time debugging and looking for relevant information across the docs, API reference, github issues, github discussion and sometimes taking pdb for a help. 

I think it would be a nicer and easier route by having something like a Pandas to Vaex cheatsheet 📖  around. 

It could be built by the community (partially extracted from issues) and reviewed by maintainers. It could also advocate best practices on places, where there are more ways how to do it. Also, if a new feature is around, it could be updated over time.

What are your thoughts?

If it would help motivate the cheatsheet creation, I can also mention things, that I had troubles with and how I combated them at the end (most probably not in the most efficient manner).

**Comment by Hasham04** [comment]
I am willing to contribute to this.

**Reply by MichalChromcak** [reply]
thanks🙏 same here :)

**Reply by JovanVeljanoski** [reply]
It is a very nice idea, it has been proposed in the past i think (loooong time ago).

I do not think I have that much free time to invest in this, but if you want to start something up and make a PR with some ideas etc... we can definitely help you out with making suggestions more efficient or what we consider "best practices".

**Reply by MichalChromcak** [reply]
Have you got any preference on the format? Part of docs? (could be harder to update for some people, but might benefit from MRs) Some specific (maybe a bit curated) place in the discussion, where conversations can happen?

**Reply by Hasham04** [reply]
I think a pinned discussion or conversation would be ideal.

**Reply by JovanVeljanoski** [reply]
I do not know exactly what you have in mind.. perhaps some guide in the docs would be nice? Maybe some table that explains the two approaches?

I fear a pinned discussion  is open to all kind of comments and might be hard to follow eventually.. 

I understand a PR to the docs might be less accessible to some people.. and does require work to maintain..

Any opinions @maartenbreddels ?

**Reply by maartenbreddels** [reply]
What about a notebook in https://vaex.io/docs/guides.html for when people come from pandas?
I do worry about maintenance, but if they are general enough.. i think it makes sense
